documentation/manual/src/main/asciidoc/modules/infinispan.asciidoc

```diff
@@ -1014,87 +1020,92 @@ configure the caches for the most important data to have a synchronous CacheStor
 or none at all, to favour performance over redundancy.
 
-==== Caches creation and Configuration
+[[infinispan-remote-cache-configuration]]
+
+==== Caches creation and configuration
 
-The Hibernate ORM property ``hibernate.ogm.datastore.create_database`` is now supported by the dialect.
-When this property is enabled the dialect will create any missing caches on the remote Infinispan cluster.
+Hibernate OGM can create a cache on the remote Infinispan cluster if it doesn't exist.
+You can enable this behavior by setting the property
+``hibernate.ogm.datastore.create_database=true``.
 
-By default all caches created use the __OGM default configuration__:
+By default, the creation of a new cache is done using the following __default configuration__:
+
+.Hibernate OGM default configuration for new Infinispan caches
 ====
 [source, XML]
 ----
-<infinispan><cache-container>
-  <distributed-cache-configuration name="configuration">
-     <locking striping="false" acquire-timeout="10000" concurrency-level="50" isolation="REPEATABLE_READ"/>
-     <transaction locking="PESSIMISTIC" mode="%s" /> # <1>
-     <expiration max-idle="-1" />
-     <indexing index="NONE" />
-     <state-transfer timeout="480000" await-initial-transfer="true" />
-  </distributed-cache-configuration>
-</cache-container></infinispan>"
+<infinispan>
+  <cache-container>
+    <distributed-cache-configuration name="configuration">
+      <locking striping="false" acquire-timeout="10000" concurrency-level="50" isolation="REPEATABLE_READ"/>
+      <transaction locking="PESSIMISTIC" mode="%s" /> # <1>
+      <expiration max-idle="-1" />
+      <indexing index="NONE" />
+      <state-transfer timeout="480000" await-initial-transfer="true" />
+    </distributed-cache-configuration>
+  </cache-container>
+</infinispan>
 ----
-<1> Transaction mode attribute will be filled with the value of the propery `hibernate.ogm.cache.transaction.mode`
+<1> Transaction mode attribute will be filled with the value of the property `hibernate.ogm.cache.transaction.mode`
 ====
 
-To apply a custom configuration to a newly created cache
-we need a previously defined cache or an existing __cache configuration__ on the remote Infinispan server to use as templates.
-
-There are two alternative ways to apply a cache configuration to new caches:
+If a different default configuration is defined on the Infinispan server, it's possible to use
+the property ``hibernate.ogm.datastore.cache_configuration`` to refer to it.
+This means that Hibernate OGM will create new caches using the selected configuration.
 
-* Using the dialect property: ``hibernate.ogm.datastore.cache_configuration``
-* Using the ``@CacheConfiguration`` annotation on Entity class
+If one some of your entities need a different configuration, you can use the annotation 
+``@CacheConfiguration``. This annotation will override the behavior of the global property
+and use the configuration defined for the annotated entity.
```

Comment:
s/use/will use, or even better `using`, I think

Reply:
Ok for `using`